cunumeric/coverage.py

```diff
@@ -60,8 +60,8 @@ def filter_namespace(
 
 
 # todo: (bev) use callback protocol type starting with 3.8
-def implemented(func: Any, prefix: str) -> Any:
-    name = f"{prefix}.{func.__name__}"
```

Comment:
Can't rely on `__name__` since `binary_ufunc`, etc do not have

Reply:
I can add that property if that's necessary.